src/Packetery/Module/Checkout.php

```diff
@@ -1416,7 +1416,7 @@ public function filterPaymentGateways( array $availableGateways ): array {
 		// phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```

Comment:
PacketSubmitter řádek 260

Tam je to samé odchytávání jako v getByWcOrderWithValidCarrier, takže bych to podle toho upravil

Reply:
TO samé prakticky je na ApiExtender řádek 68

Reply:
Good point, k tomu ty metody jsou.